automation/test.sh

```diff
@@ -46,6 +46,7 @@ elif [[ $TARGET =~ cnao ]]; then
   export KUBEVIRT_PROVIDER=${TARGET/-cnao/}
   export KUBEVIRT_DEPLOY_CDI=false
 elif [[ $TARGET =~ sig-network ]]; then
+  export KUBEVIRT_WITH_MULTUS_V3="${KUBEVIRT_WITH_MULTUS_V3:-true}"
```

Comment:
Shouldn't you check it is not `sig-network-multus-v4`?

Reply:
Not necessarily, by having `KUBEVIRT_WITH_MULTUS_V3` have a default value (`true`) it's possible to override it in the multus-v4 job (by specifying `KUBEVIRT_WITH_MULTUS_V3=false`) regardless of the `TARGET` name. 
https://github.com/kubevirt/project-infra/pull/2641/commits/081263c2720b5cd29dba1d151c435c255539e78f#diff-26f7cd0389085c1da47f919a02c4dbeca5ddd96ca00d8e49c0862f698475aaf9R1170-R1173

I went that way because I didn't like to add another `if` to this very long script.